deploy/manifests/kip/base/statefulset.yaml

```diff
@@ -1,35 +1,35 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: virtual-kubelet
+  name: kip
   labels:
-    app: virtual-kubelet
+    app: kip
 spec:
   ports:
   - port: 80
-    name: virtual-kubelet
+    name: kip
   clusterIP: None
   selector:
-    app: virtual-kubelet
+    app: kip
 ---
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
-  name: virtual-kubelet
+  name: kip
   namespace: kube-system
   labels:
-    app: virtual-kubelet
+    app: kip
 spec:
-  serviceName: virtual-kubelet
+  serviceName: kip
   replicas: 1
   selector:
     matchLabels:
-      app: virtual-kubelet
+      app: kip
   template:
     metadata:
       labels:
-        app: virtual-kubelet
-        virtual-kubelet: ""
+        app: kip
+        kip: ""
```

Comment:
Not related to this PR, but I think we can remove the `kip: ""` label and use `app: kip` for the pod anti-affinity match below. Adding `kip: ""` is redundant, since we only use it for pod anti-affinity.